client/src/routes/account/components/programForm/form.js

```diff
@@ -288,6 +288,15 @@ class ProgramForm extends React.Component {
                                      value={values.yearLevels}
                                      options={this.optionsYearLevels}
                   />
+                  <input type="checkbox" className="form-check form-check-inline" onClick={(event) => {
```

Comment:
This change belongs to https://github.com/CodeforAustralia/cese-snippet/pull/164 and shouldn't be part of the PR.